components/network/samba/Makefile

```diff
@@ -156,10 +160,12 @@ REQUIRED_PACKAGES += library/jansson
 REQUIRED_PACKAGES += library/libarchive
 REQUIRED_PACKAGES += library/ncurses
 REQUIRED_PACKAGES += library/openldap
+REQUIRED_PACKAGES += library/perl-5/parse-yapp-524
```

Comment:
Should it be in BUILD_DEPENDENCIES section?

Reply:
Due to https://wiki.samba.org/index.php/Package_Dependencies_Required_to_Build_Samba#Mandatory

It should only be necessary for building. Same for perl. I changed that in the Makefile.